<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pulse easy settings tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/iskn.png</normaloff>:/img/iskn.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="groupBox1">
      <property name="title">
       <string>1. Settings file location</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="settingsPath">
         <property name="text">
          <string>eg: app_directory/bin/data/settings.xml</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="settingsPathButton">
         <property name="text">
          <string>Choose settings file...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select the settings file you wish to edit (should be located inside the &quot;data&quot; directory next to the binary of the program): </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>2. App configuration</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Song</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Path to song (relative to the &quot;data&quot; folder): </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="songPath">
              <property name="text">
               <string>eg: music/happy.mp3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="songPathButton">
              <property name="text">
               <string>Choose audio file...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Tempo: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="tempo">
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>120</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Volume (between 0 and 1):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="volume">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Display</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Width: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="width">
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>1600</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Height: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="height">
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>900</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Display mode: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="displayMode">
              <property name="minimumSize">
               <size>
                <width>76</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Fullscreen</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Window</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Enable background (disabling the background can increase the app speed): </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="background">
              <item>
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>No</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Debug</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Replace Slate input by mouse (if you don't have a Slate this can be pretty useful): </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="debugSlate">
              <item>
               <property name="text">
                <string>No</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Play a sound on each beat during level1 (to see if your tempo value is good and if the song starts in phase): </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="debugTempo">
              <item>
               <property name="text">
                <string>No</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>3. Save/load</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>103</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="load">
         <property name="text">
          <string>Load settings from file</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/load.png</normaloff>:/img/load.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>102</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>Save settings to file</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>103</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exit">
        <property name="text">
         <string>Exit</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
